### PR TITLE
Slight rework of the event admin screen

### LINKED ISSUE #485: Some eventadmin ordering bugs
### Describe the bug

- Ordering by name looks random: this is supposedly by name ascending. 
IMG_8504

- You can order by payment even on free events.
- "Precense" in the filter popup should be "Presence".

### PR DESCRIPTION
**Pull request by JAicewizard**
Closes #485.

### Summary
This rework fixed a couple issues:
 - spelling
 - ordering (and filtering) for payment type on free events
 -  name ordering does not work

### How to test
Steps to test the changes you made:
1. Go to scarletto event
2. Check that ordering/filtering no longer shows payment things
3. Check name sorting works

### CODE REVIEW

File: lib/ui/screens/event_admin_screen.dart [hunk 1]
```diff
@@ -21,34 +21,38 @@ class EventAdminScreen extends StatefulWidget {
...
-                false: true,
-              },
-              title: 'Precense',
-              asString: (item) => item ? 'Is present' : 'Is not present',
-              toKey: (item) => item.present),
+            map: {
+              true: true,
+              false: true,
+            },
+            title: 'Precence',
```
Comment (DeD1rk) [inline lib/ui/screens/event_admin_screen.dart:42]:
```suggestion
            title: 'Presence',
```

**Comment by DeD1rk** [comment]
It still doesnt quite sort alphabetically: I get a few random items after which it gets sortedimage

**Comment by DeD1rk** [comment]
I guess it's sorting by displayname i stead of firstnamd

**Comment by JAicewizard** [comment]
I remember I changed that because there was an issue with sorting by name, but it was a while ago

**Comment by JAicewizard** [comment]
Can you rereview? This is good otherwise

**Review by DeD1rk — APPROVED** [review]

**Pull request merged**